runtime/environment-cpu.yml

```diff
@@ -16,6 +16,10 @@ dependencies:
   - scikit-learn=1.1.2
   - scipy=1.9.1
   - tqdm=4.64.1
+  - imbalanced-learn=0.10.0
+  - xgboost=1.7.1
   - pip:
       - flwr[simulation]==1.1.0
       - tensorflow-cpu==2.10.0
+      - phe==1.5.0
+      - cryptography==38.0.4
```

Comment:
@JimKing100 and @Questions1: It looks like both of you are proposing adding the cryptography library, but different versions. (See #5)

Based on my understanding, the versions are not as far apart as the numbering suggests, because cryptography had changed their versioning scheme.  

Do either of you have some flexibility on which version is needed?